SINGLE_CONNECTION.md

```diff
@@ -0,0 +1,195 @@
+clientInfo=info | Arbitrary information about the connecting client. This is available in the CLIENTINFO column of the SYSTEM.CONNECTIONS and SYSTEM.LOCALCONNECTIONS system tables.
+clientProcessID=id | The process ID (pid) of the connecting client. The clientProcessId is available in the CLIENTPROCESSID column of the SYSTEM.CONNECTIONS and SYSTEM.LOCALCONNECTIONS system tables.
+direct=[ true \| false ] | Connect directly to TE(s) rather than via the AP(s). If not specified, direct defaults to false and connections are made via the AP(s) specified in the connection URL. When direct=true, the client connection is made directly with a TE. The hosts and port numbers in the connection URL are assumed to specify the hosts and port numbers of TE processes. The use of the direct connection property is supported for nuosql and the JDBC, C, C++, and Go drivers. When using nuosql, it supports an equivalent --direct option. Where possible, direct connection should be made using the load-balancing capability of the APs. This connection property is useful when connecting external SQL applications directly to TEs that are running on a different network - such as in the Cloud, on Kubernetes clusters, and in other containerized deployments (such as Docker) or via SSH tunneling.
+idle-timeout=number | The maximum time (in seconds) which indicates how long idle connections are left open. Although the IDLE_CONNECTION_TIMEOUT system property is a global setting for timing out idle connections, you can use this connection property to override the system property on a per connection basis. By default, idle-timeout is set to -1, meaning that the global setting is used. If you set a value greater than 0, this property overrides any value set for IDLE_CONNECTION_TIMEOUT. You may also set a value of 0 to specifically disable this property for the connection. Any time a client connection is terminated for being idle for too long, a message will be logged under the net category.
+isolation=level | The default transaction isolation level for the connection to use. Isolation levels supported are consistent_read and read_committed. Note that these values are case sensitive.
+keepAlive=[ true \| false ] | Allows enabling of TCP KeepAlive on the connection from client to Transaction Engine. If not specified, keepAlive defaults to false. When keepAlive=true, the driver enables TCP KeepAlive on the socket that it opens to the Transaction Engine. 
+LBPolicy=valueA | From the client application, this property allows selective load balancing across Transaction Engines. For information on using LBPolicy, see Load Balancer Policies. 
+LBQuery=\<selector>(\<filter>) | From the client application, this property allows selective load balancing across Transaction Engines. For information on using LBQuery, see Load Balancer Policies.
```

Comment:
"For information on using LBQuery, see Load Balancer Policies."
can be removed

Reply:
Or just add a link to the doc.